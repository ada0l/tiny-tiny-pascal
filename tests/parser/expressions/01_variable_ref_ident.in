var a: Integer = 10;
begin writeln(a); end.
