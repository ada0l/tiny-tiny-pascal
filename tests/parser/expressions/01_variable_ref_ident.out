main
   program name:
      without name
   decls:
      vars:
         var
            a 
            type:
               Integer 
            10
   block:
      compounds statements
         writeln 
            a 